index.js

```diff
@@ -18,13 +20,13 @@
 
 function paginate(query, options, callback) {
   query = query || {};
-  options = Object.assign({}, paginate.options, options);
-  let select = options.select;
+  options = options || {};
```

Comment:
@Jokero There was nothing ever sent into paginate.options and that which was wasn't being tested. It could be added if we had it in the docs and tests for it ... feel free to push :)